src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -28,11 +44,17 @@ export class PropertyTemplateOutline extends Component {
     return classNames
   }
 
-  handleAddClick = (event) => {
+  handleAddClick = (resourceTemplate) => (event) => {
     event.preventDefault()
+    const propertyTypeRows = Object.assign([], this.state.propertyTypeRow)
+    const output = AddResourceTemplate(resourceTemplate, this.props.reduxPath)
+    output.map((row) => {
```

Comment:
Is this concatenating two iterators? Is there an advantage to iterating here, vs.:

```javascript
const newPropertyTypeRows = propertyTypeRows.concat(outout)
```

Reply:
Adjusted in latest commit.